website/content/docs/release-notes/consul/v1_19_x.mdx

```diff
@@ -11,7 +11,19 @@ We are pleased to announce the following Consul updates.
 
 ## Release highlights
 
-- TBD
+- **Argo Rollouts**: Argo Rollouts can progressively deploy new versions of applications into Consul service mesh on Kubernetes using the new Argo Rollouts plugin. You can incrementally release and test new versions of applications and rollback to previous versions by splitting traffic between subsets of services. For more information, refer to [Argo rollouts](/consul/docs/k8s/deployment-configurations/argo-rollouts-configuration).
```

Comment:
```suggestion
- **Argo Rollouts**: Argo Rollouts can progressively deploy new versions of applications into Consul service mesh on Kubernetes using the new Argo Rollouts plugin. You can incrementally release and test new versions of applications and roll back to previous versions by splitting traffic between subsets of services. For more information, refer to [Argo rollouts](/consul/docs/k8s/deployment-configurations/argo-rollouts-configuration).
```
Nit: roll back instead of rollback